packages/shared/routes/onboarding/views/storage-protection-setup/views/SetupPinProtectionView.svelte

```diff
@@ -33,6 +39,15 @@
 
     async function onSetPinClick(): Promise<void> {
         await initialisePincodeManager(setPinInput)
+
+        const canInitialiseFirstShimmerClaimingAccount =
```

Comment:
Why do we do this here?

Reply:
This is so that there is an account generated by the time the claiming view displays. It feels bad right now because there's no spinner...

It could be placed elsewhere but this way it is in one place rather than two (then three when Ledger comes into the mix).